src/models/contact.js

```diff
@@ -398,6 +398,7 @@ export default class Contact {
 		if (orderKey && n && !isEmpty(n)) {
 			switch (orderKey) {
 			case 'firstName':
+			case 'phoneticFirstName':
```

Comment:
Not needed?

Reply:
Hmm. Yes. It is not absolutely necessary.
I will revert it.
```suggestion
```